src/sms-command/FormKeyValueParser/useUpdateCommandMutation.js

```diff
@@ -77,7 +77,30 @@ const updateKeyValueParserMutation = {
 export const useUpdateCommandMutation = ({ id, onAfterChange }) => {
     const engine = useDataEngine()
     const onSubmit = (values) => {
-        const variables = { ...values, id }
+        const variables = {
+            ...values,
+            id,
+            smsCodes:
```

Comment:
Same as above, seems like this could've been extracted to a utility-function. Understand if this is due to time-pressure and want to get it in regardless though.